app/panel/utils/utils.js

```diff
@@ -218,10 +218,11 @@ export function setTheme(doc, name, account) {
 	// First remove all other style elements which may be there
 	const styleList = doc.head.getElementsByTagName('style');
 	// Other kinds of loops are not supported equally across browsers
+	let themeStyle = null;
 	for (let i = 0; i < styleList.length; i++) {
 		const style = styleList[i];
 		if (style.title.startsWith(styleTitlePrefix)) {
-			doc.head.removeChild(style);
+			themeStyle = style;
 		}
 	}
```

Comment:
We can remove this comment and replace with the tidier `forEach`, which is now supported in all our supported browser versions

Reply:
Good to know, changed to a forEach loop

Reply:
I stand corrected, we're unable to use a regular forEach loop because getElementsByTagName returns an HTML collection. I've changed it to `Array,prototype.forEach.call(styleList, (style) =>...` as suggested by the following stackoverflow thread unless you think keeping a basic for loop is cleaner. Thoughts?

https://stackoverflow.com/questions/39797101/why-cant-i-use-array-foreach-on-a-collection-of-javascript-elements

Reply:
Ah dang, right, sorry. We can use `Array.from(styleList).forEach`